core/src/main/java/uk/gov/justice/services/core/dispatcher/DispatcherCache.java

```diff
@@ -23,7 +24,16 @@
 
     private final Map<DispatcherKey, Dispatcher> dispatcherMap = new ConcurrentHashMap<>();
 
-    private final DispatcherFactory dispatcherFactory = new DispatcherFactory();
+    private DispatcherFactory dispatcherFactory;
+
+    @Inject
+    public DispatcherCache(final DispatcherFactory dispatcherFactory) {
```

Comment:
Our default approach to injection is to use package scoped fields, not construction injection

Reply:
As agreed, keep these for now as have to refactor lots of other tests